test/fake_crm_mon.sh

```diff
@@ -84,7 +84,8 @@ cat <<EOF
     </node_history>
     <tickets>
     </tickets>
-    <bans>
+      <bans>
+        <ban id="cli-ban-msl_SAPHana_PRD_HDB00-on-damadog-hana01" resource="msl_SAPHana_PRD_HDB00" node="damadog-hana01" weight="-1000000" master_only="false" />
```

Comment:
this is unreleated to this pr but it will be usefull for next metric. we can keep it . is not arm

Reply:
no problemo